### ISSUE TITLE
error while loading media folders `java.lang.IllegalStateException`

### ISSUE THREAD

**Issue by augugusto** [nextcloud/android#10273]
### Cause of error
```java
Exception in thread "main" java.lang.IllegalStateException: FragmentManager has been destroyed
    at androidx.fragment.app.FragmentManager.enqueueAction(FragmentManager.java:1585)
    at androidx.fragment.app.BackStackRecord.commitInternal(BackStackRecord.java:341)
    at androidx.fragment.app.BackStackRecord.commit(BackStackRecord.java:306)
    at androidx.fragment.app.DialogFragment.show(DialogFragment.java:278)
    at com.owncloud.android.ui.activity.SyncedFoldersActivity.onSyncFolderSettingsClick(SyncedFoldersActivity.kt:571)
    at com.owncloud.android.ui.activity.SyncedFoldersActivity$load$1$1.invokeSuspend(SyncedFoldersActivity.kt:315)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
    at android.os.Handler.handleCallback(Handler.java:938)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:223)
    at android.app.ActivityThread.main(ActivityThread.java:7665)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:594)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30200190`
* Build flavor: `generic`

### Device information
* Brand: `Xiaomi`
* Device: `willow`
* Model: `Redmi Note 8T`
* Id: `RQ3A.211001.001`
* Product: `willow`

### Firmware
* SDK: `30`
* Release: `11`
* Incremental: `11470`

**Comment by AlvaroBrey** [comment]
Duplicate of #10099, but I'll keep this one as it's better formatted

**Comment by joshtrichards** [comment]
Fixed in #13075
